packages/core/src/App/Components/Elements/LiveChat/use-livechat.ts

```diff
@@ -2,11 +2,16 @@ import { useCallback, useEffect, useState } from 'react';
 import { useHistory } from 'react-router';
 import { liveChatInitialization } from './live-chat-initializer';
 import Cookies from 'js-cookie';
-import { deriv_urls } from '@deriv/shared';
+import { deriv_urls, getActionFromUrl } from '@deriv/shared';
 import { useIsMounted } from 'usehooks-ts';
 
 // Todo: Should break this into smaller hooks or utility functions.
 const useLiveChat = (has_cookie_account = false, active_loginid?: string) => {
+    const url_query_string = window.location.search;
+    const url_params = new URLSearchParams(url_query_string);
+    const reset_password = getActionFromUrl() === 'reset_password';
+    const should_not_run_livechat = url_params.get('code') && reset_password;
```

Comment:
can we rename `should_not_run_livechat` to something like `should_disable_livechat`